controllers/resources/authconfig.go

```diff
@@ -84,9 +84,10 @@ func (s *staticTemplateLoader) Load(ctx context.Context, authType AuthType, key
 	}
 
 	templateData := map[string]interface{}{
-		"Namespace":      key.Namespace,
-		"Audiences":      getAuthAudience(),
-		"AuthorinoLabel": authKey + ": " + authVal,
+		"Namespace":            key.Namespace,
+		"Audiences":            getAuthAudience(),
+		"AuthorinoLabel":       authKey + ": " + authVal,
+		"InferenceServiceName": key.Name,
```

Comment:
What is the role of this new value?

Reply:
Limit the scope of the auth check. Check that you have access to this specific InferenceService

Reply:
Thanks.